packages/datepicker/src/react/text-input-field.tsx

```diff
@@ -0,0 +1,58 @@
+import Field from '@pluralsight/ps-design-system-field'
+import { CalendarIcon } from '@pluralsight/ps-design-system-icon'
+import { useUniqueId } from '@pluralsight/ps-design-system-util'
+import React, { ComponentProps, FC } from 'react'
+
+interface TextInputFieldProps extends ComponentProps<typeof Field> {
+  onChange?: React.ChangeEventHandler<HTMLInputElement>
+  placeholder?: string
+  value?: React.ReactText
+}
+
+export const TextInputField: FC<TextInputFieldProps> = props => {
+  let {
+    disabled,
+    label,
+    onChange,
+    onClick,
+    placeholder,
+    prefix,
+    subLabel,
+    value,
+    ...rest
+  } = props
+  const labelId = useUniqueId('text-input__label-')
+  const inputId = useUniqueId('text-input__input-')
```

Comment:
how are teams going to mock this out in snapshot tests? not something i expect you to address since we have this issue in other places - just mentioning